### DISCUSSION TITLE
Commit via VSCode GUI [Windows] [fnm]

### DISCUSSION THREAD

**Discussion by Julian-B90** [evilmartians/lefthook#885] [Q&A]
I manage my node version with fnm (i installed with winget).

I have a .lefthookrc
```shell
# ~/.lefthookrc

# An nvm way
export NVM_DIR="$HOME/.nvm"
[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"

# An fnm way
export FNM_DIR="$HOME/.fnm"
[ -s "$FNM_DIR/fnm.sh" ] && \. "$FNM_DIR/fnm.sh"
```

But when i commit with VSCode Git GUI i get this error: /usr/bin/env: 'node': No such file or directory

**Comment by mrexox** [comment]
Do you have `rc` option configured in your `lefthook.yml` config?
